example/getting_started/example1/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+project(powsybl_iidm4cpp_getting_started_1)
```

Comment:
To respect the project name convention, use `powsybl-iidm4cpp-getting-started-1`

I think you should also (to be discussed):
- run clang-tidy on the examples
- maybe add an option to compile the sample (disable by default)
- add the example path in the doxygen file